os/drivers/serial/serial.c

```diff
@@ -125,6 +139,86 @@ static const struct file_operations g_serialops = {
  * Private Functions
  ************************************************************************************/
 
+#ifdef CONFIG_DYNAMIC_LOGS_STORING
+void read_log_net(void)
```

Comment:
If this is a private function, pls add startic modifier. Else move it under public functions.